core/src/main/resources/hudson/model/SlaveComputerConfiguration/global.jelly

```diff
@@ -0,0 +1,10 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
+
+  <f:section title="Remoting" name="remoting">
+      <f:entry title="Reject connection from unsupported remoting versions," field="rejectConnection">
```

Comment:
Strings should be localizable. E.g. `${%Remoting}` or `${%Reject connection from unsupported remoting versions}`. Comma in the title should be removed IMHO